packages/performance/src/services/perf_logger.ts

```diff
@@ -107,6 +107,10 @@ export function logTrace(trace: Trace): void {
   if (!settingsService.dataCollectionEnabled && !trace.isAuto) {
     return;
   }
+  // Do not log if required apis are not available.
+  if (!Api.getInstance().requiredApisAvailable()) {
```

Comment:
Should we also add these "if condition" to logNetworkRequest() in this file?

Reply:
No need, the trace api can be used to create custom trace even if the sdk is not initialized, but there is no public api for networks.

Reply:
Got it, suggest to use console logger to log that required APIs are not available for the scenario above.